compiler/shared/main/scala/mlscript/compiler/mono/specializer/Specializer.scala

```diff
@@ -7,7 +7,7 @@ import mlscript.compiler.mono.MonomorphError
 import mlscript.compiler.mono.Monomorph
 import mlscript.TypeName
 import mlscript.{App, Asc, Assign, Bind, Blk, Bra, CaseOf, Lam, Let, Lit,
-                 New, Rcd, Sel, Subs, Term, Test, Tup, With, Var, Fld, FldFlags, If, PolyType, 
+                 New, NuNew, Rcd, Sel, Subs, Term, Test, Tup, With, Var, Fld, FldFlags, If, PolyType, 
```

Comment:
This whole slab of imports is really completely silly.

Just make the package declaration:

```
package mlscript
package compiler
package mono
```
(the `specializer` sub-package seems excessive)

Then you won't need any of these silly imports.